utils/docker-utils.sh

```diff
@@ -230,16 +231,22 @@ docker::deploy_service() {
         config::set_config_digests "${DOCKER_COMPOSE_PATH}/$DOCKER_COMPOSE_FILE"
     fi
 
+    # Adding Dev compose file to the params
     if [[ -n "${DOCKER_COMPOSE_DEV_FILE}" ]]; then
-        docker_compose_dev="-c ${DOCKER_COMPOSE_PATH}/$DOCKER_COMPOSE_DEV_FILE"
+        docker_compose_param="-c ${DOCKER_COMPOSE_PATH}/$DOCKER_COMPOSE_DEV_FILE"
+    fi
+
+    # Adding Dev mount compose file to the params
+    if [[ -n "${DOCKER_COMPOSE_DEV_MOUNT}" ]]; then
+        docker_compose_param="$docker_compose_param -c ${DOCKER_COMPOSE_PATH}/$DOCKER_COMPOSE_DEV_MOUNT"
     fi
```

Comment:
Let's do if else, and invert the conditions, wdyt ?

Reply:
they can both exist, we can have DEV and DEV_MOUNT compose files